### DISCUSSION TITLE
Paramétrage liste déroulante

### DISCUSSION THREAD

**Discussion by bperan29** [dbarzin/mercator#610] [Q&A] [answered]
Bonjour.
Tout d'abord merci pour cet outil qui va beaucoup nous aider.
Je suis totalement débutant et ai commencé par la vue physique et je bloque sur les listes déroulantes.
Exemple ; le type de serveur physique, le nom de cluster, le système d'exploitation.
J'ai peut-être loupé quelque chose. Toutes mes excuses si c'est le cas.
Merci d'avance pour votre aide.

**Comment by bperan29** [comment]
Ok pour le Cluster Vue logique.

**Comment by dbarzin** [comment]
Le contenu des listes déroulante va se compléter avec les données que vous y mettez. La liste déroulante contiendra ce qui se trouve dans les autres lignes de la table correspondante.

**Reply by bperan29** [reply]
Merci pour votre retour mais je n'y arrive pas. Par exemple pour un serveur logique, au niveau de l'infrastructure logique, pour le système d'exploitation cela ne me propose que UBUNTU 20.04 et je ne vois pas où (dans quelle table) renseigner un nouvel OS.

**Reply by bperan29** [reply]
J'ai peut-être trouvé. Je pense que c'est parce que j'étais en granularité niveau 3

**Comment by dbarzin** [comment, marked as answer]
Il suffit d'entrer la nouvelle valeur dans le champ de recherche et de pousser <<enter>>.

![image](https://github.com/dbarzin/mercator/assets/43208232/98875357-a85a-4872-80c2-5052525c6315)
